api/conf/settings/logging.py

```diff
@@ -115,6 +115,8 @@ def health_check_filter(record: LogRecord) -> bool:
 
     if not DEBUG:
         # WARNING: Do not run in production long-term as it can impact performance.
-        MIDDLEWARE.append(
-            "api.middleware.force_debug_cursor_middleware.force_debug_cursor_middleware"  # noqa: E501
+        middleware = (
+            "api.middleware.force_debug_cursor_middleware.force_debug_cursor_middleware"
         )
+        if middleware not in MIDDLEWARE:
+            MIDDLEWARE.append(middleware)
```

Comment:
Based on the comment, did we actually want to remove this?

Reply:
I didn't try to understand or change this bit. It was added by @zackkrida in #945.

Reply:
Yeah this enables the middleware which lets us log all DB queries, so we should keep it!